### ISSUE TITLE
Browser search within page can find content that's not visible

### ISSUE THREAD

**Issue by ppnehe** [kubernetes/website#41469]
https://kubernetes.io/docs/reference/kubectl/cheatsheet/

When I search this page using keyword ingress or Ingress I do get there are 6 results but the results are not highlighted in Yellow color . This problem is not seen when I search with other keywords such as Deployment or deployment or services or any other word .

Kindly confirm whether you are also seeing this problem and what needs to be done to resolve this.

I am using latest chrome browser.
<img width="973" alt="Screenshot 2023-06-05 at 07 59 40" src="https://github.com/kubernetes/website/assets/58841702/3cfc918c-5e6e-40fb-89d8-d1a7f4a98d8b">

**Comment by T-Lakshmi** [comment]
@ppnehe,
Yes, same results when i search also. Seems search is working properly for **ingress** or **Ingress**.

**Comment by tamilselvan1102** [comment]
Chrome browser is finding and highlighting but in the keyword(ingress or Ingress) are there deep in the side menu. It is not visible because side menu will not expand automatically.
It is not a bug.
![image](https://github.com/kubernetes/website/assets/20103020/642cfc87-2ba0-41e0-a7af-5ee4df6373df)

**Comment by sftim** [comment]
Suggestion for a fix: https://github.com/kubernetes/website/issues/45062#issuecomment-1935906808
